xbin.sh

```diff
@@ -1,6 +1,7 @@
 function xbin() {
   command="$1"
-  args="${*:2}"
+  shift
+  args="$@"
```

Comment:
```suggestion
  args=( "$@" )
```

https://github.com/koalaman/shellcheck/wiki/SC2124